tracer/tracer.go

```diff
@@ -0,0 +1,67 @@
+package tracer
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"log/slog"
+	"strings"
+	"time"
+
+	"github.com/jackc/pgx/v5"
+)
+
+type ctxKey string
+
+type Tracer struct {
+	Logger          *slog.Logger
+	LogSqlStatement bool
+	// replace placeholders with arguments
+	ReplacePlaceholders bool
+	// maybe we should not log the parameters on production because of GDPR ??
+	IncludeParams bool
```

Comment:
LogValues bool

Reply:
By default, we get `SELECT * FROM ... WHERE x=? AND y=?`.

If enabled, we'll start getting values .. `SELECT * FROM ... WHERE x=10 AND y=20`